packages/graphql/tests/schema/aggregations.test.ts

```diff
@@ -733,6 +733,10 @@ describe("Aggregations", () => {
             }
 
             input PostLikesConnectFieldInput {
+              \\"\\"\\"
+              Whether or not to create a duplicate of relationship if it already exists, instead of just updating any properties.
+              \\"\\"\\"
+              asDuplicate: Boolean
```

Comment:
Great to see a description! 👏 

I would personally assign this a default value (which is currently `false`) to make it clear from the schema what the current behaviour is. This default value should be changed by the directive argument when this is eventually implemented.

Reply:
I actually had it that way to begin with but I changed it to work better with the second part of this (the `@relationship` arg). If this is never undefined then it's not possible to know when the default behaviour defined on the directive should be overwritten